src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -410,3 +401,27 @@ func enableBorda(bordaReportInterval time.Duration, bordaSamplePercentage float6
 
 	ops.RegisterReporter(reporter)
 }
+
+func includeInSample(deviceID string, samplePercentage float64) bool {
+	if samplePercentage == 0 {
+		return false
+	}
+
+	// Sample a subset of device IDs.
+	// DeviceID is expected to be a Base64 encoded 48-bit (6 byte) MAC address
+	deviceIDBytes, base64Err := base64.StdEncoding.DecodeString(deviceID)
```

Comment:
@oxtoacart it strikes me that at least for sampling purposes we don't care about the correlation of this device ID with the device ID elsewhere in the app -- i.e. we could just use `uuid.NodeID()` directly here and skip decoding. The only reason I mention it is things might get weird if for some reason the encoding changed without realizing there's an assumption about the encoding here.

Additionally, though, I do wonder about storing the device ID from a data retention perspective. Are we really using it at all? If not, I'd be tempted to also remove that from the data we store.